Web/src/app/shared/footer/footer.component.html

```diff
@@ -1,9 +1,11 @@
 <footer>
   <div class="container">
-    <a class="logo-font" routerLink="/">The Comic Shop</a>
-    <span class="attribution">
-      &copy; {{ today | date: 'yyyy' }}.
-      An interactive learning project from <a href="https://springhrough.com">Springthrough</a>.
-    </span>
+    <div class="d-flex justify-content-center">
+      <a class="logo-font" routerLink="/">The Comic Shop </a>
+      <span class="attribution">
+        &copy; {{ today | date: 'yyyy' }}.
+        An interactive learning project from <a href="https://springhrough.com">Springthrough</a>.
```

Comment:
Looks like I introduced a typo on springthrough.com in the href. We'll want to fix that soon!

Reply:
heh, didn't notice that before. Nice catch!